docs/source/notebooks/table_of_contents_tutorials.js

```diff
@@ -13,7 +13,7 @@ Gallery.contents = {
     "ODE_API_shapes_and_benchmarking": "Deep dives",
     "DEMetropolisZ_EfficiencyComparison": "Deep dives",
     "DEMetropolisZ_tune_drop_fraction": "Deep dives",
-    "PyMC3_tips_and_heuristic": "How-To",
+    "conditional-autoregressive-model": "How-To",
```

Comment:
Could you move this from "How-to" to Gaussian Process?

Reply:
done